subworkflows/nf-core/utils_nfcore_pipeline/tests/main.function.nf.test

```diff
@@ -0,0 +1,134 @@
+
+nextflow_function {
+
+    name "Test Functions"
+    script "../main.nf"
+    config "subworkflows/nf-core/utils_nfcore_pipeline/tests/nextflow.config"
+    tag "subworkflows"
+    tag "subworkflows_nfcore"
+    tag "utils_nfcore_pipeline"
+    tag "subworkflows/utils_nfcore_pipeline"
+
+    test("Test Function checkConfigProvided") {
+
+        function "checkConfigProvided"
+
+        then {
+            assertAll(
+                { assert function.success },
+                { assert snapshot(function.result).match() }
+            )
+        }
+    }
+
+    test("Test Function checkProfileProvided") {
+
+        function "checkProfileProvided"
+
+        when {
+            function {
+                """
+                input[0] = []
```

Comment:
[] supposed to be empty?

Reply:
Hmmm... that does seem odd, but I guess so! This is an nf-core one so I'm scared to change it haha